CHANGELOG.md

```diff
@@ -113,13 +113,21 @@ Main (unreleased)
 
 - Added an `exclude_event_message` option to `loki.source.windowsevent` in flow mode,
   which excludes the human-friendly event message from Windows event logs. (@ptodev)
+
+- Make component list sortable in web UI. (@hainenber)
+
+- Added an `exclude_event_message` option to `loki.source.windowsevent` in flow mode,
+  which excludes the human-friendly event message from Windows event logs. (@ptodev)
+
```

Comment:
duplicated entries to remove

Reply:
Duplicate entry removed. Thanks for pointing this one out!